python/ql/test/library-tests/regex/FirstLast.expected

```diff
@@ -86,30 +86,30 @@
 | ^[A-Z_]+$(?<!not-this) | last | 1 | 7 |
 | ^[A-Z_]+$(?<!not-this) | last | 1 | 8 |
 | ^[A-Z_]+$(?<!not-this) | last | 8 | 9 |
-| ^[A-Z_]+$(?<!not-this) | last | 20 | 21 |
+| ^[A-Z_]+$(?<!not-this) | last | 13 | 21 |
 | ax{01,3} | first | 0 | 1 |
 | ax{01,3} | last | 1 | 2 |
 | ax{01,3} | last | 1 | 8 |
-| ax{01,3} | last | 7 | 8 |
```

Comment:
This (and others like it) looks like an existing bug. I think it is due to [this line](https://github.com/github/codeql/blob/main/python/ql/lib/semmle/python/regex.qll#L414) in `simpleCharacter` not considering qualifiers longer than one character (such as the multiples notation `{n, m}`).